core/metacling/src/CMakeLists.txt

```diff
@@ -59,6 +52,13 @@ ROOT_OBJECT_LIBRARY(MetaCling
   TClingTypedefInfo.cxx
   TClingTypeInfo.cxx
   TClingValue.cxx
+  INCLUDES
+     ../res
```

Comment:
the same here..I was curious if there is a way not to use relative paths here?

Reply:
Also here one can specify absolute path like `${CMAKE_SOURCE_DIR}/core/metacling/res`
Actually these are several places where I just type fast to test - now I will prefer to use absolute pathes

Reply:
these will be amazing!